error_test.go

```diff
@@ -59,6 +59,67 @@ func newMultiErr(errors ...error) error {
 	return &multiError{errors: errors}
 }
 
+func TestEvery(t *testing.T) {
+	myError1 := errors.New("woeful misfortune")
+	myError2 := errors.New("worrisome travesty")
+
+	for _, tt := range []struct {
+		desc       string
+		giveErr    error
+		giveTarget error
+		wantIs     bool
+		wantEvery  bool
+	}{
+		{
+			desc:       "all match",
+			giveErr:    newMultiErr(myError1, myError1, myError1),
```

Comment:
can you add a case under error_post_go120_test.go where you do this with an error created with errors.Join?